unified-scheduler-pool/src/lib.rs

```diff
@@ -1351,7 +1338,7 @@ pub trait SpawnableScheduler<TH: TaskHandler>: InstalledScheduler {
 
     fn spawn(
         pool: Arc<SchedulerPool<Self, TH>>,
-        initial_context: SchedulingContext,
```

Comment:
I don't think `initial_context` is adding any substential value for readability. so, preferred consistency by naming this as just `context`.

Reply:
otherwise, i'd have to name `result_with_timings` as `initial_result_with_timings`, which is mouthful, imo.